src/main/js/main_html5.js

```diff
@@ -418,6 +413,19 @@ require("../../../html5-common/js/utils/environment.js");
     };
 
     /**
+     * Prepares a video element to be played via API.  This is called on a user click event, and is used in
```

Comment:
that's a good question.  The reason we haven't needed one is because it has always been so fast that events were never raised in our testing.  I can implement the block on events as well